### ISSUE TITLE
Revamped 7z releases

### ISSUE THREAD

**Issue by NightYoshi370** [DS-Homebrew/TWiLightMenu#918]
Hey, I'm not sure how to follow the format on this one because it doesn't frustrate me personally and there is no alternative.
So, I feel like there's currently a bit too many steps in the TWiLight Menu++ extraction process. Even though everything is straight forward, I've seen people just extract everything to the root without reading the folders, and then requiring support about it.

Doesn't help that the dumb video guide just omits a step and since everyone follows that to the dot, they don't do that step. Then, they ask support for it.

Obviously, people should just read what they're doing but we could make it better on ourselves.
Why not make any non-conflicting item be right there? For example, the 3DS CFW firmware doesn't really need to be its own folder. Same for DSi&3DS - SD card users. Just put that out.

Also, when you have a specific roms folder, it makes people think that it has to follow that structure, when TWiLight could navigate anywhere.

TL;DR: We don't need to separate everything for things that don't conflict with eachother. For the users that want to have a clean SD card, just have a readme saying which folder applies to what and if it's not for them, then they can delete

**Comment by SleepyLark** [comment]
I think a better idea would be to have separate .7z files with everything sorted out so people can just extract straight to the root like "Twilight Menu (3DS).7z", "Twilight Menu (DSi).7z", "Twilight Menu (DSi)(NO CFW).7z", the only problem would be B4DS since there's multiple flashcarts also the issue of having to organize this for each release.

**Comment by LabregoPT** [comment]
Hi, I didn't know of this so I oppened #971. As I stated there, it'd be nice to include optional features such as widescreen patches and several different themes on a separate download, thus reducing the time of extraction and installation, either on PC and via the Updater

**Comment by NightYoshi370** [comment]
So, if we don't end up doing SleepyLark's suggestion, I still suggest the following: https://github.com/DS-Homebrew/TWiLightMenu/tree/master/7zfile/_nds/TWiLightMenu/widescreen
These files are in the root level _nds folder instead of the `3DS - CFW users` folder. This means that it is included in every TWiLight Menu++ install, even for Flashcard and DSi Users.

We should move it to the 3DS folder.
